src/containers/builder/builder.container.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, Icon } from 'antd';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { StatusTag } from '../../common/components/status-tag/status-tag.component';
+import { Survey } from '../../common/types/survey.type';
+import { AppState } from '../../state';
+import './builder.container.less';
+
+export interface BuilderRouteInfo {
+    surveyid: string;
+}
+
+export interface BuilderContainerProps {
+    survey: Survey;
+}
+
+const mapStateToProps = (state: AppState): Partial<BuilderContainerProps> => ({
+    survey: state.surveysState.surveys.find(s => s.id === state.router.location.pathname.split('/').reverse()[0]),
+});
+
+// @ts-ignore
+@connect(mapStateToProps)
+export class BuilderContainer extends Component<BuilderContainerProps> {
+    render() {
+        const { survey } = this.props;
+
+        if (!survey) {
+            return <span>Survey not found</span>;
```

Comment:
I will turn this into an sfc so we can reuse it

Reply:
Sure, let's just make a general 404 component, can be simple for now

Reply:
👍